packages/firebase_inappmessaging/lib/firebase_inappmessaging.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class FirebaseInAppMessaging {
+  static const MethodChannel _channel =
```

Comment:
You'll probably want to call this `channel` and make it `@visibleForTesting` so that you can write unit tests.

Reply:
done!